torchaudio/models/wavernn.py

```diff
@@ -324,3 +347,27 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def get_pretrained_wavernn(checkpoint_name: str, progress: bool = True) -> WaveRNN:
+    r"""Get pretrained WaveRNN model.
+
+    Here are the available checkpoints:
+
+    - wavernn_10k_epochs_8bits_ljspeech
+
+        WaveRNN model trained with 10k epochs and 8 bits depth waveform on the LJSpeech dataset.
+        The model is trained using the default parameters and code of the examples/pipeline_wavernn/main.py.
+
+    Args:
+        checkpoint_name (str): The name of the checkpoint to load.
+        progress (bool): If True, displays a progress bar of the download to stderr.
+    """
+    if checkpoint_name in model_config_and_urls:
+        url, configs = model_config_and_urls[checkpoint_name]
+        model = WaveRNN(**configs)
+        state_dict = load_state_dict_from_url(url, progress=progress)
+        model.load_state_dict(state_dict)
+        return model
+    else:
+        raise ValueError("The model_name `{}` is not supported.".format(checkpoint_name))
```

Comment:
The argument is `checkpoint_name` so I think it's better to stick to it.

Reply:
Thanks for the comment.
It is addressed [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R371).